config.yaml

```
@@ -57,20 +57,34 @@ params:
   about:
     enable: true
     title: About Us
```

```suggestion
    title: About us
```
I will be better to stick to normal case. `params.hero.title` also needs to be changed to 'Welcome to PythonPune'.